src/bin/historical_batch.rs

```diff
@@ -0,0 +1,100 @@
+use anyhow::ensure;
+use ethportal_api::{consensus::beacon_state::HistoricalBatch, utils::bytes::hex_encode};
+use portal_bridge::types::era::Era;
+use regex::Regex;
+use ssz::Encode;
+use std::{
+    fs::{File, OpenOptions},
+    io,
+    io::{BufRead, Write},
+};
+use surf::get;
+use tree_hash::TreeHash;
+
+const _BELLATRIX_SLOT: u64 = 4700013;
+const _CAPELLA_SLOT: u64 = 6209536;
+
+const ERA_URL: &str = "https://mainnet.era.nimbus.team/";
+
+#[tokio::main]
+async fn main() -> anyhow::Result<()> {
+    let download_dir = dirs::download_dir().unwrap();
+    let file_path = download_dir.join("historical_batches");
+
+    let body = get(ERA_URL).recv_string().await.unwrap();
+    // Match era files with epoch between 00573 and 00759
+    let re =
+        Regex::new(r#"".*((0057[3-9])|(005[8-9][0-9])|006[0-9][0-9]|(007[0-5][0-9])).*\.era""#)
```

Comment:
I would probably like to see some tests for this regex, but imo these scripts are a little less "official" than core code, so i'll leave it up to you